docs/more-pages.md

```diff
@@ -51,6 +51,18 @@ Create the `_sidebar.md`:
 
 You need to create a `.nojekyll` in `./docs` to prevent GitHub Pages from ignoring files that begin with an underscore.
 
+!> Docsify only looks for `_sidebar.md` in the current folder, and uses that, otherwise it falls back to the one configured using `$docsify` config.
```

Comment:
```suggestion
!> Docsify only looks for `_sidebar.md` in the current folder, and uses that, otherwise it falls back to the one configured using `window.$docsify.loadSidebar` config.
```

May be this can be more detailed way ?